devices/merrytek.js

```diff
@@ -0,0 +1,141 @@
+                return {
+                    occupancy: value > 0 ? true : false,
+                };
+            case tuya.dataPoints.mrytSensitivity:
+                return {
+                    sensitivity: value,
+                };
+            case tuya.dataPoints.mrytIlluminanceLux:
+                return {
+                    illuminance_lux: value,
+                };
+            case tuya.dataPoints.mrytIlluminanceCalibration:
+                return {
+                    illuminance_calibration: value,
+                };
+            case tuya.dataPoints.mrytKeepTime:
+                return {
+                    keep_time: value,
+                };
+            case tuya.dataPoints.mrytLED_Enable:
+                return {
+                    led_enable: value == 1 ? true : false,
+                };
+            case tuya.dataPoints.mrytMotionState:
+                return {
+                    states: lookup[value],
+                };
+            case 102:
+                return {
+                    LuxChangeTrigger: value,
+                };
+            case 106:
+                return {
+                    HoldTime: value,
+                };
+            case 108:
+                return {
+                    Scene: value,
+                };
+            case 110:
+                return {
+                    Minormotion: value,
+                };
+            case 111:
+                return {
+                    Breathing: value,
+                };
+            case 112:
+                return {
+                    Linght: value,
+                };
+            case 113:
+                return {
+                    Behavior: value,
+                };
+            default:
+                meta.logger.warn(`zigbee-herdsman-converters:tuya_smart_vibration_sensor: NOT RECOGNIZED ` +
+                        `DP #${dpValue.dp} with data ${JSON.stringify(dpValue)}`);
+            }
+        },
+    },
+};
+
+module.exports = [
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_9xejegcy'}],
+        model: 'MSA201_Z',
+        vendor: 'Merrytek',
+        description: 'Large beam microwave occupancy sensor 24GHz',
+        fromZigbee: [fzLocal.merrytek_microwave_sensor],
+        toZigbee: [tzLocal.merrytek_microwave_sensor],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic']);
+        },
+        exposes: [
+            e.occupancy(),
+            exposes.numeric('states', ea.STATE).withDescription('Motion state')
+                .withDescription('No motion, big motion, minor motion, breathing'),
+            e.illuminance_lux(),
+            exposes.binary('led_enable', ea.STATE_SET, true, false).withDescription('Enabled LED'),
+            exposes.enum('keep_time', ea.STATE_SET, ['0', '1', '2', '3', '4', '5', '6', '7'])
```

Comment:
Please change this to the enum with the real values, so instead of `0`, `1`; `30s`, `180s` and do a mapping in the converter.

Reply:
Sorry I cannot do this, it is above my current skill level. Could you help me on this?

The basis of this was a different product from the same manufacturer, so I have just replicated the same approach in Z2M